packages/client/src/types.ts

```diff
@@ -9,7 +9,8 @@ export interface Project {
 export interface ProjectEntities {
   get(id: string): Promise<WrappedEntity>;
   query(query?: Query): Promise<WrappedEntity[]>;
-  create(entity: any): Promise<WrappedEntity>;
+  create(entity: Entity): Promise<WrappedEntity>;
+  patch(entity: Entity): Promise<WrappedEntity>;
```

Comment:
I think `update` is a better name for this method than `patch`

Reply:
Done